apps/discussions/package.json

```diff
@@ -33,7 +33,8 @@
     "eslint-plugin-standard": "^4.0.0",
     "prettier": "^1.17.1",
     "react-hot-loader": "^4.8.8",
-    "solidity-coverage": "0.5.11"
+    "solidity-coverage": "0.5.11",
+    "solium": "1.1.8"
```

Comment:
How did these `discussions` changes sneak into this PR?

Reply:
Probably we merged `dev` branch here at some point to advance on conflict solving, but it should not count as commit anyway... good catch, cleaning it.